### ISSUE TITLE
builds for apple silicon

### ISSUE THREAD

**Issue by knaaptime** [conda-forge/pandoc-crossref-feedstock#54]
on new M1 macs, you can install pandoc-crossref form homebrew, but there's no pre-built binary in the releases of the [official package](https://github.com/lierdakil/pandoc-crossref/releases/) (might be worth crossrefrencing there)

releases over there are built using gh-actions, and it's not clear when gha will [add support](https://github.com/actions/virtual-environments/issues/2187) for apple silicon builds.

since conda-forge now has good [support for M1](https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/), we could try and rig up a recipe here that builds from source. But that would probably not be trivial

**Comment by ickc** [comment]
We should keep this open to track this.

However, pandoc is yet to be built from source in conda-forge: https://github.com/conda-forge/pandoc-feedstock/issues/54

So probably this should be done after the above issue is resolved.

On the other hand, could we just grab the pre-built binaries from homebrew? I think I looked into this earlier and find that it needs some sort of authentication (forget the details at the moment.) But I'm sure it can be done in some way. But is that acceptable?

**Comment by ickc** [comment]
c.f. https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fpandoc-crossref

**Comment by mlund** [comment]
An arm64 build has been requested here: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1181.

**Comment by mfansler** [comment]
Technically, the feedstock is now generating **osx-arm64** "builds", however, I'd advocate this Issue should remain open until actual *native* builds are shipping. AFAIK, the current recipe is a compromise: it ships the x86_64 binary under the **osx-arm64** subdir and assumes that Rosetta2 will seamlessly emulate.

**Comment by knaaptime** [comment]
agree. also i think the only pandoc build for arm64 at the moment is v2.12 and pandoc-crossref requires >=2.18
